program_transformers/src/token_metadata/v1_asset.rs

```diff
@@ -411,3 +418,31 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
 
     Ok(Some(DownloadMetadataInfo::new(mint_pubkey_vec, uri)))
 }
+
+async fn upsert_asset_v1_account_attachments<T: ConnectionTrait + TransactionTrait>(
+    conn: &T,
+    mint_pubkey: &Pubkey,
+    slot: u64,
+) -> ProgramTransformerResult<()> {
+    let edition_pubkey = MasterEdition::find_pda(mint_pubkey).0;
+    let mint_pubkey_vec = mint_pubkey.to_bytes().to_vec();
+    let attachment = asset_v1_account_attachments::ActiveModel {
+        id: ActiveValue::Set(edition_pubkey.to_bytes().to_vec()),
+        asset_id: ActiveValue::Set(Some(mint_pubkey_vec.clone())),
+        slot_updated: ActiveValue::Set(slot as i64),
+        // by default, the attachment type is MasterEditionV1
+        attachment_type: ActiveValue::Set(V1AccountAttachments::MasterEditionV1),
```

Comment:
You could probably assume `MasterEditionV2` instead as way more of them are V2 now.  Although I know its planned to be updated by the other program transformer handler anyways.

Unless did you have some specific reason for choosing V1 as a default?  V2 is a subset of V1, but doesn't seem like that would matter here.

Reply:
no specific reason , will set it to MasterEditionV2 as default